diff --git a/chrome/browser/profiles/profile_io_data.cc b/chrome/browser/profiles/profile_io_data.cc
index 3bf4c0fd66c185e839f29c3b54543d6dd21a550b..4f2ccc5a187eb43a4a31423a02698be1f8a3aa36 100644
--- a/chrome/browser/profiles/profile_io_data.cc
+++ b/chrome/browser/profiles/profile_io_data.cc
@@ -791,6 +791,7 @@ void ProfileIOData::InstallProtocolHandlers(
 void ProfileIOData::AddProtocolHandlersToBuilder(
     net::URLRequestContextBuilder* builder,
     content::ProtocolHandlerMap* protocol_handlers) {
+  protocol_handlers->insert(std::make_pair("magnet", std::make_unique<MagnetProtocolHandler>()));
   for (auto& protocol_handler : *protocol_handlers) {
     builder->SetProtocolHandler(protocol_handler.first,
                                 std::move(protocol_handler.second));
@@ -1088,7 +1089,7 @@ void ProfileIOData::Init(
         std::make_unique<network::URLRequestContextBuilderMojo>();
 
     std::unique_ptr<ChromeNetworkDelegate> chrome_network_delegate(
-        new ChromeNetworkDelegate(
+        new BraveProfileNetworkDelegate(
 #if BUILDFLAG(ENABLE_EXTENSIONS)
             io_thread_globals->extension_event_router_forwarder.get()));
 #else
