diff --git a/chrome/browser/ui/browser_command_controller.cc b/chrome/browser/ui/browser_command_controller.cc
index 9b551948c9a71f4221ae1182291aed0bb10101f4..e11444482f04efdb453a2c5aea33b317d0a3c6ad 100644
--- a/chrome/browser/ui/browser_command_controller.cc
+++ b/chrome/browser/ui/browser_command_controller.cc
@@ -943,6 +943,7 @@ void BrowserCommandController::InitCommandState() {
   UpdateCommandsForContentRestrictionState();
   UpdateCommandsForBookmarkEditing();
   UpdateCommandsForIncognitoAvailability();
+  UpdateCommandsForTor();
 }
 
 // static
@@ -992,6 +993,7 @@ void BrowserCommandController::UpdateCommandsForIncognitoAvailability() {
     command_updater_.UpdateCommandEnabled(IDC_IMPORT_SETTINGS, false);
     command_updater_.UpdateCommandEnabled(IDC_OPTIONS, false);
   }
+  UpdateCommandsForTor();
 }
 
 void BrowserCommandController::UpdateCommandsForTabState() {
