diff --git a/chrome/browser/ui/webui/settings/md_settings_ui.cc b/chrome/browser/ui/webui/settings/md_settings_ui.cc
index e2219baeb53fc667679e9173a1d3ecefee412a7b..acc8dda55f64a6807b5364821dbbc11f8b6e7a58 100644
--- a/chrome/browser/ui/webui/settings/md_settings_ui.cc
+++ b/chrome/browser/ui/webui/settings/md_settings_ui.cc
@@ -136,6 +136,8 @@ void MdSettingsUI::RegisterProfilePrefs(
   registry->RegisterBooleanPref(prefs::kImportDialogHistory, true);
   registry->RegisterBooleanPref(prefs::kImportDialogSavedPasswords, true);
   registry->RegisterBooleanPref(prefs::kImportDialogSearchEngine, true);
+  registry->RegisterBooleanPref(prefs::kImportDialogCookies, true);
+  registry->RegisterBooleanPref(prefs::kImportDialogStats, true);
 }
 
 MdSettingsUI::MdSettingsUI(content::WebUI* web_ui)
