diff --git a/chrome/common/pref_names.h b/chrome/common/pref_names.h
index 2baccfde79b3aa41a081f9783b17ef73913251f2..13f5b1d41aa13fa4b8be474536440ad42ab087d5 100644
--- a/chrome/common/pref_names.h
+++ b/chrome/common/pref_names.h
@@ -361,6 +361,8 @@ extern const char kImportDialogBookmarks[];
 extern const char kImportDialogHistory[];
 extern const char kImportDialogSavedPasswords[];
 extern const char kImportDialogSearchEngine[];
+extern const char kImportDialogCookies[];
+extern const char kImportDialogStats[];
 
 extern const char kProfileAvatarIndex[];
 extern const char kProfileLocalAvatarIndex[];
