diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
index a2374002d61b2d655fb320544b902490b1a87f43..274851c0a6440a77792edcb17aafc16dc2fcf8fe 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
@@ -199,6 +199,7 @@ base::string16 AvatarToolbarButton::GetAvatarTooltipText() const {
   if (IsIncognito())
     return l10n_util::GetStringUTF16(IDS_AVATAR_BUTTON_INCOGNITO_TOOLTIP);
 
+  if (profile_->IsTorProfile()) return l10n_util::GetStringUTF16(IDS_TOR_PROFILE_NAME);
   if (profile_->IsGuestSession())
     return l10n_util::GetStringUTF16(IDS_GUEST_PROFILE_NAME);
 
