diff --git a/chrome/common/pref_names.cc b/chrome/common/pref_names.cc
index eb6ffcf47514756b9ab2bbae08a4117492c22004..2c299f50bc327dacc2fa4eadf830817a92931fc2 100644
--- a/chrome/common/pref_names.cc
+++ b/chrome/common/pref_names.cc
@@ -1089,6 +1089,8 @@ const char kImportDialogBookmarks[] = "import_dialog_bookmarks";
 const char kImportDialogHistory[] = "import_dialog_history";
 const char kImportDialogSavedPasswords[] = "import_dialog_saved_passwords";
 const char kImportDialogSearchEngine[] = "import_dialog_search_engine";
+const char kImportDialogCookies[] = "import_dialog_cookies";
+const char kImportDialogStats[] = "import_dialog_stats";
 
 // Profile avatar and name
 const char kProfileAvatarIndex[] = "profile.avatar_index";
