diff --git a/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc b/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
index 96239d3f454d48c4699a7c8e7568536c04709f43..49754944dbfda8fccd48fd1c24b8c5644aca884e 100644
--- a/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
+++ b/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
@@ -2740,6 +2740,7 @@ void AddLocalizedStrings(content::WebUIDataSource* html_source,
 #endif
   policy_indicator::AddLocalizedStrings(html_source);
 
+  BraveAddLocalizedStrings(html_source, profile);
   html_source->SetJsonPath(kLocalizedStringsFile);
 }
 
