diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index dc22933980e63004cfbc2378e92d5c450663179b..c03f20c3cf9ee29aed901c70a9f914d9e318a0cb 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -250,7 +250,7 @@ grit("google_chrome_strings") {
 }
 
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs = [
@@ -454,6 +454,7 @@ service_manifest("chrome_content_packaged_services_manifest_overlay") {
       "//ui/accessibility:manifest",
     ]
   }
+  packaged_services += [ "//brave/utility/tor:tor_launcher_manifest" ]
 }
 
 # Per-profile services.
