diff --git a/chrome/browser/profiles/profiles_state.cc b/chrome/browser/profiles/profiles_state.cc
index 40d7b4d93ab305c423f11fa315ef270c99ad6817..71e67d74ff866bf31eac517c0a8a1d5334815715 100644
--- a/chrome/browser/profiles/profiles_state.cc
+++ b/chrome/browser/profiles/profiles_state.cc
@@ -84,6 +84,8 @@ base::string16 GetAvatarNameForProfile(const base::FilePath& profile_path) {
 
   if (profile_path == ProfileManager::GetGuestProfilePath()) {
     display_name = l10n_util::GetStringUTF16(IDS_GUEST_PROFILE_NAME);
+  } else if (profile_path == BraveProfileManager::GetTorProfilePath()) {
+    display_name = l10n_util::GetStringUTF16(IDS_TOR_PROFILE_NAME);
   } else {
     ProfileAttributesStorage& storage =
         g_browser_process->profile_manager()->GetProfileAttributesStorage();
