diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 52ab34ca40a7a22332d66e1650c2ec8954c98a4a..216f51411638f6c01aa82cf2c3af6a8bd109be11 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -1955,6 +1955,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
